packages/react-components/src/components/RemoteVideoTile.tsx

```diff
@@ -111,6 +114,20 @@ export const _RemoteVideoTile = React.memo(
         /* @conditional-compile-remove(one-to-n-calling) */
         /* @conditional-compile-remove(PSTN-calls) */
         participantState={props.participantState}
+        isPinned={props.isPinned}
+        onClickPinIcon={() => {
+          props.onUnpinParticipant?.();
+        }}
+        menuItems={[
```

Comment:
Open Question:
How do we make this work with the drawer menu on mobile?
Should we refrain from using a Fluent type here and introduce a simplified structure that works with both the contextual menu and the drawer menu?